CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ Unreleased changes template.
 
 {#v0-0-0-fixed}
 ### Fixed
+* (pypi) Platform specific markers are now correctly handled when using `requirments_by_platform`
+  in pip.parse Fixed in [#2766](https://github.com/bazel-contrib/rules_python/pull/2766).
```

Comment:
```suggestion
* (pypi) Platform specific extras are now correctly handled when using
  universal lock files with environment markers. Fixes [#2690](https://github.com/bazel-contrib/rules_python/pull/2690).
```

The breakage of the `env` markers has never been released so referencing the original issue would be more correct.